### PR TITLE
Add Inspektor Gadget on AKS scenario

### PR DESCRIPTION
**Pull request by blanquicet**
Hi @mbifeld, here a quick draft for an Inspektor Gadget scenario. I tested it in my machine and in the [Azure Cloud Shell](https://ms.portal.azure.com/#cloudshell/), and it worked just fine. 

BTW, I used `time` to measure how much time it takes for bringing up the cluster and deploy an Inspektor Gadget and it was around 4 minutes in the [Azure Cloud Shell](https://ms.portal.azure.com/#cloudshell/):

```bash
$ git clone https://github.com/blanquicet/InnovationEngine.git
[...]
$ cd InnovationEngine/
$ git checkout jose/ig-demo 
[...]
$ make build-ie
[...]
$ az account set --subscription <name or ID>
$ time ./bin/ie execute scenarios/ocd/DeployIGonAKS/README.md
Quickstart: Deploy Inspektor Gadget in an Azure Kubernetes Service cluster

[...]

real    3m50.690s
user    0m5.597s
sys     0m1.861s
```

### CODE REVIEW

File: scenarios/ocd/DeployIGonAKS/README.md [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+    ```
+
+## Install Inspektor Gadget
+
+Inspektor Gadget is composed of a kubectl plugin executed in the user's system and a DaemonSet deployed in the cluster.
+
+1. Install the latest version of the kubectl plugin from the releases page, uncompress and move the `kubectl-gadget` executable to `$HOME/.local/bin`:
+
+    ```bash
+    IG_VERSION=$(curl -s https://api.github.com/repos/inspektor-gadget/inspektor-gadget/releases/latest | jq -r .tag_name)
```
[the remark this answers is not in the document]
Comment (blanquicet) [reply scenarios/ocd/DeployIGonAKS/README.md:89]: Mmmm I tried and it was:

```bash
[...]
Your Cloud Shell session will be ephemeral so no files or system changes will persist beyond your current session.
jose [ ~ ]$ jq
jq - commandline JSON processor [version 1.6]

Usage:  jq [options] <jq filter> [file...]
        jq [options] --args <jq filter> [strings...]
        jq [options] --jsonargs <jq filter> [JSON_TEXTS...]

[...]
jose [ ~ ]$ curl -s https://api.github.com/repos/inspektor-gadget/inspektor-gadget/releases/latest | jq -r .tag_name
v0.22.0
```

Am I missing something?

**Review by mbifeld — APPROVED** [review]
Approved, although note that 'main' is now the primary branch so I suggest changing the target.  Also, in the one-click deploy mode, deleting at the end isn't recommended as it doesn't allow the user to use/see what was created.  But for proof of concept, it looks good!

Tested on CloudShell, executes successfully.

**Comment by blanquicet** [comment]
Hi @mbifeld, I rebased the PR, improved the description and tried to use the expected similarity with regex to match better the expected outputs. However, it is not working as I expected. I'll debug it tomorrow and I'll let you know once it is ready again for a final review.

**Review by mbifeld — APPROVED** [review]
Approved.  I highly recommend adding additional expected-similarity blocks in the future.  The version check at the end will ensure that the whole scenario succeeded, but earlier commands should have checks too so that the script doesn't continue to run if a command in the middle fails.

**Pull request merged**